moma_sensor_tools/README.md

```diff
@@ -1,13 +1,90 @@
+
+**Prerequisites:**
+- Checkerboard calibration pattern
+
+**Method:**
+The calibration of the intrinsics is done using the [`camera_calibration`](http://wiki.ros.org/camera_calibration) package, using a pinhole model.
+
+1. Run the RealSense camera stream, for example by issuing the following command on the PC with the camera attached to it:
+   ```bash
+   roslaunch moma_sensor_tools calibration.launch operator_pc:=false
+   ```
+2. Run the intrinsic calibration GUI. Make sure that the square count and size matches the checkerboard pattern that you use:
+   ```bash
+   rosrun camera_calibration cameracalibrator.py --size 7x6 --square 0.07 image:=/hand_eye/color/image_raw camera:=/hand_eye/color --no-service-check
+   ```
+3. Follow the GUI instructions, until all direction and rotation axes light up green.
+4. The calibration GUI generates a file `/tmp/calibrationdata.tar.gz` which needs to be extracted:
+   ```bash
+   tar -xvf /tmp/calibrationdata.tar.gz
+   ```
+
+5. The contained `ost.yaml` file needs to be moved to `~/calibration/camera_calibration.yaml`:
+   ```bash
+   mv /tmp/ost.yaml ~/calibration/camera_calibration.yaml
+   ```
+   If you want to keep the calibration file permanently and to share it with other users of the MoMa repo, also make a copy of it to `$(moma_sensor_tools)/config/camera_calibration/(camera-name/serialnum/identifier).yaml`. Otherwise, just keep it locally.
+
+### 2. Extrinsic (hand-eye) calibration routine
+
+![](https://github.com/IFL-CAMP/easy_handeye/raw/master/docs/img/eye_on_hand_aruco_pic.png)
+
+**Prerequisites:**
+- Apriltag marker (you can print them for example from [this resource](resources/apriltags_36h11_1-50.pdf))
+- Having completed a proper intrinsic calibration routine
+
+**Method:**
+The hand-eye calibration can be performed using the [easy_hand_eye](https://github.com/IFL-CAMP/easy_handeye) package and the [easy_hand_eye rqt gui](https://github.com/IFL-CAMP/easy_handeye/tree/master/rqt_easy_handeye). Here we provide some launch and config files which should be easy to setup and reproduce for any MoMa robot. For more information, refer to the documentation of the calibration packages.  
+
+1. Ensure the [Apriltag id and size settings](config/handeye_calibration/apriltags.yaml) match your physical ones.
+2. Launch the panda robot on the robot computer:
+    ```bash
+    roslaunch moma_robot robot_pc.launch
+    ```
+3. In a ROS-networked operator PC, run the calibration routine:
+    ```bash
+    roslaunch moma_sensor_tools calibration.launch 
+    ```
+    This will launch the realsese camera driver, and a sample collection gui that will allow to alternate between different camera viewpoints and collect apriltag marker poses. The calibration routine also launches the apriltag detector which reads the specific aprilag configuration from this [config file](config/handeye_calibration/apriltags.yaml). Make sure that the information in this file reflects the apriltag you are currently using.
```

Comment:
typo: realsense

Maybe lets say that the config file needs to contain the same id and size of the printed tag, just to make it dummy-proof